contracts/extensions/MultisigPermissions.sol

```diff
@@ -0,0 +1,551 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.8.25;
+pragma experimental ABIEncoderV2;
+
+import { ColonyDataTypes, IColony } from "./../colony/IColony.sol";
+import { ColonyRoles } from "./../colony/ColonyRoles.sol";
+import { IColonyNetwork } from "./../colonyNetwork/IColonyNetwork.sol";
+import { ColonyExtensionMeta } from "./ColonyExtensionMeta.sol";
+import { ExtractCallData } from "./../common/ExtractCallData.sol";
+import { GetActionDomainSkillId } from "./../common/GetActionDomainSkillId.sol";
+import { GetActionSummary, ActionSummary } from "./../common/GetActionSummary.sol";
+
+// ignore-file-swc-108
+
+contract MultisigPermissions is
+  ColonyExtensionMeta,
+  ColonyDataTypes,
+  ExtractCallData,
+  GetActionDomainSkillId,
+  GetActionSummary
```

Comment:
Wonder if its possible to bundle both `ExtractCallData` and `GetActionDomainSkillId` into `GetActionSummary`, since that's what we actually use here.

Reply:
Yeah, these imports are totally unnecessary, so removed.